features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/RolePermissions/AdminTable/AdminTableBody.jsx

```diff
@@ -106,7 +121,6 @@ function AdminTableBody(props) {
         setSelected(newSelected);
     };
     const isSelected = (name) => selected.indexOf(name) !== -1;
-    const emptyRows = rowsPerPage - Math.min(rowsPerPage, rows.length - page * rowsPerPage);
```

Comment:
@RakhithaRR Can you please attach a screenshot , Without this , the table will be shrunk to available number of rows not showing full table

Reply:
Earlier there were 6 empty rows in the table as mentioned in the issue.
![Screenshot from 2021-04-08 17-24-22](https://user-images.githubusercontent.com/25485273/114022443-65c3ad00-988f-11eb-8503-630759aa8c5e.png)

Reply:
yes that was intentional, because when there is only 1 role this table will have only 1 row wich does not looks good

Reply:
I see. However having a large number of empty rows when the row count is low and rows per page is high also does not look that good.

Reply:
I see, but why would someone select 100 Rows per page when there are only 2-3 roles in total?  the worst cases here is having 9 empty rows when rows per page are 10, having 10 roles when rows per page is 50 like that

Reply:
Reverted this change with c0e0fb9